deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1291,20 +1292,28 @@ def callback_registry_count_entries(line):
         return match.group(1)
 
 
+def callback_key_event(line):
+    event = callback_detect_event(line)
+
+    if event is None:
+        return None
+
+    if event['data']['attributes']['type'] != 'registry_key' or event['data']['path'] == registry_ignore_path:
+        return None
+
+    return event
+
+
 def callback_value_event(line):
-    match = re.match(r'.*Sending FIM event: .*value_name.*', line)
+    event = callback_detect_event(line)
 
-    if match is not None:
-        msg = r'.*Sending FIM event: (.+)$'
-        match = re.match(msg, line)
+    if event is None:
+        return None
 
-        try:
-            if json.loads(match.group(1))['type'] == 'event':
-                return json.loads(match.group(1))
-        except (AttributeError, JSONDecodeError, KeyError):
-            pass
+    if event['data']['attributes']['type'] != 'registry_value':
+        return None
 
-    return None
+    return event
```

Comment:
```
match_event = None if event is None or event['data']['attributes']['type'] != 'registry_value' else event

return match_event
```

Reply:
Same as the previous comment.
[Commit](https://github.com/wazuh/wazuh-qa/pull/1670/commits/c5f646dba669a1facacd74025ba515b4b452bda6)